realsense_ros_camera/launch/includes/nodelet.launch.xml

```diff
@@ -42,6 +42,8 @@
   <arg name="enable_pointcloud"   default="false"/>
   <arg name="enable_sync"         default="false"/>
 
+  <arg name="base_frame_id"       default="realsense"/>
+
   <node pkg="nodelet" type="nodelet" name="$(arg manager)" args="manager" output="screen"/>
```

Comment:
If you can, following my suggestion in another comment, you could have these two `node`s into a `<group ns="$(arg namespace)">` block.